lib/dockerregistry/types.go

```diff
@@ -349,6 +349,18 @@ type PromotionContext func(
 	Tag,
 	TagOp) stream.Producer
 
+// ImageSortable uses a slice of digests instead of a map, allowing its contents
+// to be sorted.
+type ImageSortable struct {
```

Comment:
Maybe `ImageWithTags` ?  Sortable is sort-a-blah ;-)

Reply:
I like that better than ImageSortable; ACK

Reply:
How about `ImageWithDigestSlice` ?